dashboard/app/models/game.rb

```diff
@@ -141,6 +141,10 @@ def supports_sharing?
     app == TURTLE || app == FLAPPY || app == BOUNCE || app == STUDIO || app == STUDIO_EC || app == APPLAB || app == CRAFT || app == GAMELAB
   end
 
+  def sharing_filtered?
```

Comment:
Having no familiarity with the code, it wasn't immediate what this meant or would be used for. Perhaps add a comment? To confirm my understanding, for `STUDIO` apps, we may not allow a share for various reasons. For all other apps, we allow shares (regardless of presence of share failure reasons).